test/near_provider.test.js

```diff
@@ -136,36 +136,28 @@ describe('\n---- PROVIDER ----', () => {
     describe('\n---- BASIC QUERIES ----', () => {
         describe('getProtocolVersion | eth_protocolVersion', () => {
             test('returns protocol version', withWeb3(async (web) => {
-                try {
-                    // Get current version
-                    const { version: { version } } = await testNearProvider.status();
-                    const getVersion = await web.eth.getProtocolVersion();
+                // Get current version
+                const { version: { version } } = await testNearProvider.status();
+                const getVersion = await web.eth.getProtocolVersion();
 
-                    expect(typeof version).toStrictEqual('string');
-                    expect(web3.utils.toHex(version)).toStrictEqual(getVersion);
-                } catch (e) {
-                    expect(e).toBeNull();
-                    return e;
-                }
+                expect(typeof version).toStrictEqual('string');
+                expect(web3.utils.toHex(version)).toStrictEqual(getVersion);
             }));
         });
 
         describe('isSyncing | eth_syncing', () => {
-            test('returns correct type - Boolean|Object', withWeb3(async (web) => {
-                try {
-                    const keyPair = await nearlib.KeyPair.fromRandom('ed25519');
-                    const newAccount = await web._provider.account.createAccount('test.sync', keyPair.getPublicKey(), 2);
-                    expect(newAccount).toBe('object');
+            test.skip('returns correct type - Boolean|Object', withWeb3(async (web) => {
```

Comment:
meant to point this out. This test is failing. I threw a `.skip` on it. I think it's related to an issue you filed (commented on the next line --v)

Reply:
Are you still getting this as a failing test? Mine is passing.

Reply:
yah i still get it. what did you do, delete nearcore and reclone it to get rid of your bugs?

Reply:
Yep. Issue might have been fixed in the new stable version of nearcore.

Reply:
You might not need to reclone. Just update. I only did that because I wanted to get rid of any `/.near` folders or configs that might be lurking around. Oh damn just realized I'm on `master` not `stable`.

Reply:
Oh. This was fixed like hours ago in `nearcore`. Lol. Just saw this on the latest release notes: `RPC will not return "isSyncing" error.`

Also looks like there is no more `stable` branch, only `master`.

Reply:
did not work. I'll try the reclone thingy

Reply:
do you know what commit you're on?

Reply:
ok I recloned and everthing, this is still failing 4 me

Reply:
stable is gone. We use proper versioning now https://github.com/nearprotocol/nearcore/releases

Reply:
@emaG3m is this still failing for you?

Reply:
yes

Reply:
everything elsel ooks fine. gonna merge so we can work on this in a separate PR/issue.